### PR TITLE
add orai18n jar to Oracle ZHS16GBK character

### LINKED ISSUE #4774: Non supported character set (add orai18n.jar in your classpath): ZHS16GBK
using Oracle connector query Oracle with character `ZHS16GBK` will trigger the following errors:

```
SQL ERROR [67108864]: Query failed (#20200811_011326_00003_dv24r): Non supported character set (add orai18n.jar in your classpath): ZHS16GBK
  Query failed (#20200811_011326_00003_dv24r): Non supported character set (add orai18n.jar in your classpath): ZHS16GBK
  Query failed (#20200811_011326_00003_dv24r): Non supported character set (add orai18n.jar in your classpath): ZHS16GBK
    java.sql.SQLException: Non supported character set (add orai18n.jar in your classpath): ZHS16GBK
    Non supported character set (add orai18n.jar in your classpath): ZHS16GBK
```

solution:

copy `orai18n.jar` into `/usr/lib/presto/shard`  directory , and create symbol link at `/usr/lib/presto/plugin/oracle` directory

### PR DESCRIPTION
**Pull request by wgzhao**
fix #4774

### CODE REVIEW

**Comment by wgzhao** [comment]
> Could you squash commits into one and update the commit message as
> 
> `Add orai18n for Oracle globalization support`
> 
> 
> 
> Have you tried adding a test for this?

OK

**Comment by wgzhao** [comment]
> Could you squash commits into one and update the commit message as
> `Add orai18n for Oracle globalization support`
> 
> Have you tried adding a test for this?

test container's Oracle container  only use UTF8 character and can not change, I try to add some other tests

File: presto-oracle/pom.xml [hunk 1]
```diff
@@ -68,13 +68,20 @@
...
         <dependency>
             <groupId>io.airlift</groupId>
             <artifactId>log-manager</artifactId>
             <scope>runtime</scope>
         </dependency>
 
+        <dependency>
+            <groupId>com.oracle.ojdbc</groupId>
+            <artifactId>orai18n</artifactId>
+            <version>${dep.oracle.version}</version>
```
[the remark this answers is not in the document]
Comment (wendigo) [reply presto-oracle/pom.xml:81]: That won't work @ebyhr as dependencies are sorted now by pom enforcer plugin :)

[the remark this answers is not in the document]
Comment (ebyhr) [reply presto-oracle/pom.xml:81]: Oops, you are right.

**Comment by ebyhr** [comment]
Merged, thanks!

**Pull request merged**